indico/modules/rb/views/user/rooms.py

```diff
@@ -94,6 +94,25 @@ class WPRoomBookingSearchRoomsResults(WPRoomBookingBase):
     def __init__(self, rh, menu_item, **kwargs):
         self._menu_item = menu_item
         WPRoomBookingBase.__init__(self, rh, **kwargs)
+        self.addExtraJS('''
```

Comment:
:volcano:

We are NOT going to add JS to Python files in new code... Add it to a .js file or the template inside a script tag instead...